utility/utils.py

```diff
@@ -1109,35 +1109,47 @@ def generate_self_signed_certificate(subject: Dict) -> Tuple:
     )
 
 
-def fetch_build_artifacts(build, ceph_version, platform):
+def fetch_build_artifacts(build, ceph_version, platform, upstream_build):
```

Comment:
I believe there is only one place which calls this method... but still feel we should set `upstream_build=None`.